docs/installation/yum-run.md

```diff
@@ -0,0 +1,44 @@
+# Run Percona Server for MySQL
+
+Percona Server for MySQL stores the data files in `/var/lib/mysql/` by
+default. The configuration file used to manage Percona Server for MySQL is the `/etc/my.cnf`.
+
+The following commands start, provide the server status, stop the server, and restart the server.
+
+!!! note
+
+    The RHEL distributions and derivatives come with [systemd](http:/ freedesktop.org/wiki/Software/systemd/) as the default system and service manager so you can invoke all of the commands with `sytemctl` instead of `service`. Currently, both options are supported.
+
+
+* Percona Server for MySQL is not started automatically on the RHEL distributions and derivatives after installation. Start the server with the following command:
```

Comment:
Maybe it's better to add a numbered list here instead of bullets.

Reply:
redone